pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/SegmentStatusChecker.java

```diff
@@ -184,9 +184,11 @@ public void runSegmentMetrics() {
       int nReplicasExternal = -1;
       int nErrors = 0;
       int nOffline = 0;
+      int nSegments = 0;
```

Comment:
There are too many counters here, and it is not easy to understand the logic anymore. Can you please write a one-liner on what each of these counters represent? thanks